scripts/release_scripts/wrap_up_release.py

```diff
@@ -99,6 +99,37 @@ def remove_blocking_bugs_milestone_from_issues(repo):
         issue.edit(milestone=None)
 
 
+def remove_protection_rule():
```

Comment:
`ask_user_to...` (Currently, it looks like this method will remove the protection rule.)

Reply:
Done.